packages/vaadin-lumo-styles/gulpfile.js

```diff
@@ -118,8 +119,8 @@ import './version.js';
 
 const $_documentContainer = document.createElement('template');
 
-$_documentContainer.innerHTML = \`<vaadin-iconset name="lumo">
-${createIconset(folder, filenames, 'vaadin-icon:')}
+$_documentContainer.innerHTML = \`<vaadin-iconset name="lumo" size="1000">
```

Comment:
Is `size="1000"` intentional here? Is this really the default pixel size of Lumo icons?

Reply:
Yes, this is what we used to have. @jouni do you think we should change this?

Reply:
The icons are drawn on a 24x24 canvas, so that seems like the correct value instead of this. These type of values have been used in the icon font, though, to set the baseline of the icons. I admit I don’t really know the details about how that works.

So, feel free to change to 24 if it works. But I think that requires that the coordinates in the SVG paths are adjusted as well.

Reply:
As long as the `1000` value is intentional here, I see no need to change that now. It just looked strange, and I wanted to check if that's not a mistake.

Reply:
Yes that's the culprit here.

Reply:
Created #1996 so we could change this in a separate PR.